db/queries/flags/queries.py

```diff
@@ -33,9 +33,9 @@ def create_flag_for_application(
     return flag_metadata
 
 
-def retrieve_flags_for_application(application_id: str) -> dict:
+def retrieve_flags_for_application(application_ids: list[str]) -> dict:
```

Comment:
Just minor, but could this be renamed to retrieve_flags_for_application**s**() so that it's a bit more clear that it's a bulk function now?

Reply:
Good spot have added in https://github.com/communitiesuk/funding-service-design-assessment-store/pull/99/commits/d7740ce61e9a0f6a93cf9fe5681f66fc13b41016